files/ru/learn/css/css_layout/responsive_design/index.html

```diff
@@ -95,11 +95,11 @@ <h2 id="Media_Queries_Медиавыражения">Media Queries (Медиав
 } </code>
 </pre>
 
-<p>Вы можете добавлять несколько медиавыражений в пределах одной таблицы стилей, подстраивая весь ваш макет или его части так чтоб соответствовать наилучшим образом разным размерам экрана. Точки, в которых применяется media query и меняется макет, известны как <em>контрольные точки.</em></p>
+<p>Вы можете добавлять несколько медиавыражений в пределах одной таблицы стилей, подстраивая весь ваш макет или его части так, чтобы наилучшим образом соответствовать разным размерам экрана. Точки, в которых применяется медиавыражение и меняется макет, известны как <em>контрольные точки.</em></p>
 
-<p>Общим подходом при использовании Media Queriy является создание простого одно колоночного макета для устройств с узкими экранами (например, мобильные телефоны), затем проверка для больших экранов и применение макета с несколькими столбцам, когда вы знаете, что у вас достаточно ширины экрана чтобы уместить все. Это часто называют дизайном <strong>сначала мобильный</strong> (<strong>mobile first</strong>).</p>
+<p>Общим подходом при использовании медиавыражений является создание простого одноколоночного макета для устройств с узкими экранами (например, мобильные телефоны), затем проверка для больших экранов и применение макета с несколькими столбцам, когда вы знаете, что у вас достаточно ширины экрана, чтобы уместить все. Это часто называют дизайном <strong>"сначала мобильные"</strong> (<strong>mobile first</strong>).</p>
```

Comment:
```suggestion
<p>Общим подходом при использовании медиавыражений является создание простого одноколоночного макета для устройств с узкими экранами (например, мобильные телефоны), затем проверка для больших экранов и применение макета с несколькими столбцам, когда вы знаете, что у вас достаточно ширины экрана, чтобы уместить все. Такой подход часто называют <strong>mobile first</strong> дизайном.</p>
```
Не вполне уверен, насколько нужно переводить этот термин. Он очень устоявшийся именно в таком виде.

Reply:
Да тоже думал над этим. Вязл такой же перевод как в книге "Mobile first"